keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -489,24 +492,40 @@ More information on Windows resource access can be found at https://docs.microso
 
 #### Container Mounts
 
-- When privileged containers are started a new volume will be created on the host which will contain the contents of the container image. Containers will have a default working directory that points to this container volume. Containers will also have full access to the host file-system (unless restricted by filed-based ACLs and the run_as_username used to start the container.) Processes should use absolute paths when accessing files on the host and relative paths when accessing files brought in via the container image.
-Note: there will be no `chroot` equivalent.
-- An environment variable `$CONTAINER_SANDBOX_MOUNT_POINT` will be set to the absolute path where the container volume is mounted.
+- When privileged containers are started a new volume will be created on the host which will contain the contents of the container image. Containers will have a default working directory that points to this container volume.
```

Comment:
Does privileged need to be updated to Host Process here?

Reply:
There's a couple other mentions of privileged but they're outside the diff

Reply:
Hmm, I mush have missed some. Let me update.